modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,195 @@
+
+* Before this update, `TaskRun` and `CustomRun` creation could fail immediately on transient mutating-webhook timeouts, causing flakiness on busy clusters. With this update, the system  applies exponential backoff, configurable via the `wait-exponential-backoff` ConfigMap and controlled by the `enable-wait-exponential-backoff` setting:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: feature-flags
+  namespace: tekton-pipelines
+data:
+  enable-wait-exponential-backoff: "true"
+#...
+----
+
+* Before this update, the controller used a fixed thread count, limiting concurrency. With this update, you can override `threads-per-controller` by setting the `THREADS_PER_CONTROLLER` environment variable, allowing finer control over controller concurrency.
+
+* Before this update, `podTemplate` fields in `TaskRunSpec` CR did not support parameter substitution, limiting matrix and multi-arch patterns. With this update, the controller  substitutes parameters in `podTemplate` fields for all `TaskRuns` and `TaskRunSpecs`.
+
+* Before this update, the `onError` block in `pipeline` v1beta1 did not support variables. With this update, the controller  resolves the `onError` variables, improving error handling flexibility.
+
+* Before this update, git resolver shell-outs sometimes failed to inherit environment variables, breaking environment-driven authentication or configuration. With this update, the resolver passes the pod environment correctly to all git subprocesses.
+
+* Before this update, git resolver deployments could leave zombie git processes running, consuming resources. With this update, resolvers  use a `tini`-based image and entrypoint to actively clean up subprocesses.
+
+* Before this update, upgrading Tekton could prevent updates to metadata, such as finalizers, on a completed `PipelineRun` or `TaskRun` due to specification drift. With this update, you can  update metadata on completed PipelineRuns and TaskRuns immediately after upgrades.
+
+* Before this update, the OpenShift git resolver did not mount the trusted CA ConfigMap into the component system CA store, potentially causing certificate verification issues. With this update, the ConfigMap is mounted correctly, ensuring secure git operations.
+
+* Before this update, the `git-clone` task failed with a `No such remote 'origin'` error messgae if the `origin` remote was missing from the repository. With this update, the task automatically adds the `origin` remote to the repository configuration, ensuring correct setup and successful cloning.
+
+* Before this update, streaming live logs for a PipelineRun or TaskRun with the {tekton-results} CLI required the `-f` flag. Attempts to query logs without the flag resulted in errors, and status messages for running resources were unclear. With this update, the CLI streams logs without requiring the `-f` flag, and provides clearer status messages when querying logs for running `PipelineRuns` and `TaskRuns`.
```

Comment:
For some reason, this is coming inside the "pipelines" section. Which is not correct.
Also, this point is conveying the wrong information. Correct point should be:
```
Before this update, users querying logs for a running PipelineRun or TaskRun in OPC Results received an unclear error message. With this update, the message clearly indicates that logs cannot be retrieved while the run is still in progress.
```